This will be addressed in the next upstream release (>1.10) as shown here:

https://git.kernel.org/pub/scm/libs/ell/ell.git/commit/?id=3916c1cc32be205e185a8a56de5cde1b9ec60e81

--- ell/pem.c.orig	2020-11-30 09:02:25.583791998 -0500
+++ ell/pem.c	2020-11-30 09:03:23.548913496 -0500
@@ -224,7 +224,11 @@
 
 		/* Check that each header line has a key and a colon */
 		while (start < end) {
+#if defined(__GLIBC__)
 			const char *lf = rawmemchr(start, '\n');
+#else
+			const char *lf = memchr(start, '\n', end - start);
+#endif
 			const char *colon = memchr(start, ':', lf - start);
 
 			if (!colon)
